Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2698,17 +2698,17 @@ def __str__(self):
         initial_refcount = sys.getrefcount(obj)
 
         coro = coroutine_function()
-        loop = asyncio.new_event_loop()
```

Comment:
You can just add `loop.close()` at the end no? that would cause less diff

Reply:
The other tests in this file use contextlib.closing

Reply:
You can also disable whitespace changes to see a smaller diff